sos/policies/distros/redhat.py

```diff
@@ -232,6 +232,9 @@ class RHELPolicy(RedHatPolicy):
     _upload_url = RH_SFTP_HOST
     _upload_method = 'post'
     _device_token = None
+    # Max size for an http single request is 1Gb
+    _max_size_request = 1073741824
```

Comment:
This is the byte-precise granted size Red Hat Portal allows, right? (rather, one byte less due to `size >= self._max_size_request`)

I just want to prevent sporadic cases when a border-line-sized sosreport passes the check but fails to be uploaded - rather go the other way, failover to SFTP even for sizes slightly below the limit.

Reply:
Yes, that's right. I checked with a bit more than 1Gb but can check the exact size as well in case we need to change the value.